README.md

```diff
@@ -1,25 +1,104 @@
 # Scripts to manage GitHub Classroom
 
-This folder contains scripts to collect assignments and push feedback to student repositories in a given GitHub organization. There are no repo creation scripts here, since we use GitHub classroom for this part. Look at [https://github.com/snadi/UsefulCourseScripts](https://github.com/snadi/UsefulCourseScripts) if you are interested in that.
+This folder contains scripts to collect assignments or labs and push feedback to student repositories in a given GitHub organization. There are no repository creation scripts here, since we use GitHub classroom for this part. 
 
-# Prerequisites
-
-The following python libraries are being used:
+## Dependencies
 
+- Python 3
 - [github3](https://github.com/sigmavirus24/github3.py)
 
-Install packages with: `pip3 install -r requirements.txt`
+````shell
+# Quick installation steps
+pip3 install -r requirements.txt
+````
+
+These scripts have been tested with Python 3.9.1(latest).
+
+
+
+## Directory structure
+
+```
+.
+├── ccid_list.txt               # Contains ccids you need to grade
+├── collect_repos.py            # Collect repos
+├── config                      # Configration file for collect_repos.py
+├── push_feedback.sh            # Push feedback to all repos
+├── README.md                   # This file
+├── requirements.txt            # For dependencies installation
+├── student_list.csv            # A mapping from ccids to github usernames
+├── token_file.txt              # Your Github access token(You need to create this file)
+├── update_repos.sh             # Update all repos
+│
+├── assignment1                 # A subdirectory named with an assignment or lab name 						
+│   │                           # It's automatically created after running collect_repos.py
+│   ├── repo1 
+│   │   ├── feedback.txt        # Feedback you made
```

Comment:
Can we maybe change to `# Feedback you made. Could have been generated with marking scripts you had or added manually`